identity/aziot-identity-common/src/lib.rs

```diff
@@ -44,8 +44,18 @@ pub struct AuthenticationInfo {
 #[derive(Debug, Clone, serde::Deserialize, serde::Serialize)]
 #[serde(rename_all = "lowercase")]
 pub enum AuthenticationType {
-	SaS,
-	X509,
+    Sas,
+    X509,
+}
+
+impl std::fmt::Display for AuthenticationType {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let s = match *self {
+            AuthenticationType::Sas => "Sas",
+            AuthenticationType::X509 => "X509",
```

Comment:
Nit: In the HTTP payloads, these will be "sas" and "x509" because of the serde rename_all to lowercase. So perhaps the casing here should be consistent with that.

Reply:
Removed it since it wasn't used.